src/node.rs

```diff
@@ -487,11 +487,20 @@ impl PbftNode {
                 let block_id_matches = block.previous_id == wb.get_block_id();
 
                 if !block_num_matches || !block_id_matches {
-                    debug!(
-                        "Block didn't match for catchup, skipping: {} {}",
-                        block_num_matches, block_id_matches
-                    );
-                    return Ok(false);
+                    if block.block_num == head.block_num + 2 {
+                        debug!(
+                            "Block didn't match for catchup, adjusting: {} {}",
+                            block_num_matches, block_id_matches
+                        );
+                        state.working_block =
```

Comment:
Can you explain this change? This is not a logging change

Reply:
@agunde406: I was testing a fix in Jenkins for something that I couldn't reproduce locally, and I was using the verbose logging in this PR to do it. The fix seems to be working, so I've pulled it out of this PR.